backend/glp/models.py

```
@@ -0,0 +1,307 @@
+        settings.AUTH_USER_MODEL,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name='glp_reservations_claimed',
+    )
+    claimed_at = models.DateTimeField(null=True, blank=True)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=['guild_id', 'discord_id'], name='glp_reservation_unique'),
+        ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)